### ISSUE TITLE
child_process: process.send causes memory leaks

### ISSUE THREAD

**Issue by lolBig** [Rokid/ShadowNode#199]
```js
// parent.js
var child = require('child_process').fork(__dirname + '/child.js', {
  env: {
    isSubprocess: 'true',
  }
})
child.on('message', data => {
  // console.log(data.toString())
})

//child.js
setInterval(() => {
  process.send(Math.random())
}, 0)
```
child's memory increases fast

**Comment by lolBig** [comment]
caused by process.send()

**Comment by yorkie** [comment]
Could you have some statistic data to show what the problem is in details? For instance, the leaks is on parent, child or both?

**Comment by lolBig** [comment]
```
ps -o rss,args,comm | grep iotjs
  2868 iotjs /Users/qile222/Desktop/parent.js     iotjs
  2452 /usr/local/bin/iotjs /Users/qile222/Desktop/child.js     /usr/local/bin/iotjs
```

after few moments
```
ps -o rss,args,comm | grep iotjs
  2876 iotjs /Users/qile222/Desktop/parent.js     iotjs
  3324 /usr/local/bin/iotjs /Users/qile222/Desktop/child.js     /usr/local/bin/iotjs
```

**Comment by lolBig** [comment]
iotjs_string_t? haha

**Comment by yorkie** [comment]
Ok, it seems that child.js has leak? Good catch!

**Comment by yorkie** [comment]
This should be fixed at #200, @lolBig please take a look when you are free :)
